config/config-autoscaler.yaml

```diff
@@ -108,6 +108,12 @@ data:
     # observed pods.
     max-scale-up-rate: "1000.0"
 
+    # Max scale down rate limits the rate at which the autoscaler will
+    # decrease pod count. It is the maximum ratio of observed pods versus
+    # desired pods.
```

Comment:
I think an example would be useful to illustrate what `2.0` really means, e.g. 100 -> 50 (and over what period does this apply?)

Reply:
Done.